worlds/alttp/Shops.py

```diff
@@ -301,7 +301,7 @@ def create_shops(world, player: int):
                         new_items = new_dark_shop
                 keeper = world.random.choice([0xA0, 0xC1, 0xFF])
                 player_shop_table[name] = ShopData(typ, shop_id, keeper, custom, locked, new_items, sram_offset)
-    if world.mode[player] == "inverted":
+    if world.world_state[player] == 2:
```

Comment:
If the int is because importing Options breaks up top, you can just import right below `def create_shops`